content/about/_index.md

```diff
@@ -3,26 +3,30 @@ title: About
+Public health authorities (federal, state, local), fellow researchers (working on epidemic tracking and forecasting), public and private sectors, data journalists, and the general public.
 
-### What has our impact been?
+### Delphi's milestones
 
-Since 2013, we've supported CDC's Influenza Division in advancing and growing a scientific community around flu forecasting.  We've been perennial leaders in forecasting accuracy, and started providing flu nowcasts to state departments of health in 2016.
+- Since 2013, we've supported CDC's Influenza Division in advancing and growing a [scientific community around flu forecasting](https://www.cdc.gov/flu/weekly/flusight/index.html). We've been perennial leaders in forecasting accuracy.
 
-Since 2019, we've been working directly with CDC as a National Center of Excellence for Influenza Forecasting (a 5-year designation).
+- Since 2016, we've been TODO RONI please describe the Epidata API and you can [use this](https://cmu-delphi.github.io/delphi-epidata/) to link to it. 
 
-Since March 2020, we've created and maintained the nation's largest public repository of diverse, geographically-detailed, real-time indicators of COVID activity in the U.S.  Our indicators cover every rung of the severity pyramid, and they're used regularly by public health officials, DoD personnel, data journalists, healthcare companies, financial firms, fellow modelers, and COVID response groups.  We've made our indicators freely available through a public API, with R and Python packages for easy access.
+- Since 2016, we've been providing [flu nowcasts](https://delphi.cmu.edu/nowcast/) to state departments of health.
 
-Several of the underlying data sources (on which these indicators are built) would not exist or be publicly available without our efforts. This includes: 
+- Since 2019, we've been working directly with CDC as a National Center of Excellence for Influenza Forecasting (a 5-year designation).
 
-A massive national daily survey we're running in partnership with Facebook.  This has reached over 12 million Americans since April, providing real-time insights into, e.g., self-reported symptoms, mask wearing, testing, and contacts, broken down by various demographics.  Survey data on symptoms and testing can serve as early indicators of COVID activity while leapfrogging over public health reporting systems; data on mask wearing and contacts can guide policy and outreach. 
+- Since March 2020, we've created and maintained the [nation's largest public repository of diverse, geographically-detailed, real-time indicators of COVID-19 activity](https://covidcast.cmu.edu) in the U.S. Our indicators cover every rung of the [severity pyramid](https://docs.google.com/presentation/d/1jvIycxDRMEIozKIowv2UyvSqZyF5y6jR8EAXUEK22D4/edit?usp=sharing), and they're freely available through a [public API](https://cmu-delphi.github.io/delphi-epidata/api/covidcast.html).
```

Comment:
You should use "shortcodes" to manage internal and API links.  For example, COVIDCAST should point to:

`[COVIDcast map]({{< relref "covidcast" >}})`.  You can use `[Delphi group]({{< relref "/" >}})` for links to the main page.  This is so that we can follow links in non-production environments.

And the API page would be at:

`[COVIDcast API]({{< apiref "api/covidcast.html" >}})`

`

Reply:
> @sgratzl can you add a commit to this PR that fixes all of the links?

done

Reply:
Thanks! I pinged @RoniRos for his changes and then we can merge.